commands/command_pointer.go

```diff
@@ -85,7 +85,7 @@ func pointerCommand(cmd *cobra.Command, args []string) {
 		fmt.Printf("Pointer from %s\n\n", pointerName)
 
 		if err != nil {
-			Error(err.Error())
+			Error("Not a valid Git LFS pointer file.")
```

Comment:
Yeah, it is. There are a few things left that aren't quite right and need to be polished up before this is fully ready.